src/twisted/web/http.py

```diff
@@ -2427,12 +2425,7 @@ def headerReceived(self, line):
         if not self._maybeChooseTransferDecoder(header, data):
             return False
 
-        reqHeaders = self.requests[-1].requestHeaders
-        values = reqHeaders.getRawHeaders(header)
-        if values is not None:
-            values.append(data)
-        else:
-            reqHeaders.setRawHeaders(header, [data])
+        self.requests[-1].requestHeaders.addRawHeader(header, data)
```

Comment:
Thanks for the cleanup.

I guess that addRawHeader API was added later, and this is why we had the previous usage with get and set